development/tools/cmd/rendertemplates/main.go

```diff
@@ -2,15 +2,23 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"path"
+	"path/filepath"
 	"text/template"
 
 	"github.com/Masterminds/semver"
 	"github.com/Masterminds/sprig"
 	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const (
+	// autogenerationMessage is message added at the beginning of each autogenerated file.
+	autogenerationMessage = "File generated by rendertemplates. DO NOT EDIT."
```

Comment:
While you're on it. Can you adapt the message to match go standards? https://golang.org/pkg/cmd/go/internal/generate/

Reply:
At lest for the go code

Reply:
@sjanota nice, I wasn't' aware of that, I will do 👍

Reply:
ah.. I thought that you linked for some go command which will apply header for me, but this message is already based on go standards, I've only changed the Code -> File because we are adding that comment also to yaml files, In your opinion, I should have different headers for different files extensions?

Reply:
neh, let's have the same message, but based on go standards

Reply:
neh, let's have the same message, but based on go standards. I meant this comment:
```
To convey to humans and machine tools that code is generated,
generated source should have a line that matches the following
regular expression (in Go syntax):

    ^// Code generated .* DO NOT EDIT\.$
```

Reply:
thanks for the feedback, adjusted 👍